be/test/exprs/vectorized/math_functions_test.cpp

```diff
@@ -1308,6 +1308,39 @@ TEST_F(VecMathFunctionsTest, OutputNanTest) {
         }
     }
 
+    {
+        std::vector<bool> null_expect = {false, false, true};
+        std::unique_ptr<FunctionContext> ctx(FunctionContext::create_test_context());
+        ColumnPtr result = MathFunctions::sinh(ctx.get(), columns);
+        auto nullable = ColumnHelper::as_raw_column<NullableColumn>(result);
+        ASSERT_EQ(nullable->size(), null_expect.size());
+        for (size_t i = 0; i < nullable->size(); i++) {
+            ASSERT_EQ(nullable->is_null(i), null_expect[i]);
+        }
+    }
+
+    {
+        std::vector<bool> null_expect = {false, false, true};
+        std::unique_ptr<FunctionContext> ctx(FunctionContext::create_test_context());
+        ColumnPtr result = MathFunctions::cosh(ctx.get(), columns);
+        auto nullable = ColumnHelper::as_raw_column<NullableColumn>(result);
+        ASSERT_EQ(nullable->size(), null_expect.size());
+        for (size_t i = 0; i < nullable->size(); i++) {
+            ASSERT_EQ(nullable->is_null(i), null_expect[i]);
```

Comment:
Besides the values of null, the result should be checked too.

Reply:
Thanks @imay for your review. I will add the check.

Reply:
Hi @imay , I add TrigonometricFunctionTest to check the results. Please have a look. Thank you.